### PR TITLE
docs: update using-SPDK-vhostuser-and-kata.md

### PR DESCRIPTION
**Pull request by liubin**
Use `ctr` instead of `Docker`.

Fixes: #3054

Signed-off-by: bin <bin@hyper.sh>

### CODE REVIEW

File: docs/use-cases/using-SPDK-vhostuser-and-kata.md [hunk 1]
```diff
@@ -228,20 +228,41 @@ $ sudo mknod /var/run/kata-containers/vhost-user/block/devices/vhostblk0 b 241 0
...
 ## Launch a Kata container with SPDK vhost-user block device
 
-To use `vhost-user-blk` device, use Docker to pass a host `vhost-user-blk`
-device to the container. In docker, `--device=HOST-DIR:CONTAINER-DIR` is used
+To use `vhost-user-blk` device, use `ctr` to pass a host `vhost-user-blk`
+device to the container. In your `config.json`, you should use `devices`
 to pass a host device to the container.
 
-For example:
+For example(only `vhost-user-blk` listed):
```
Comment (cmaf) [inline docs/use-cases/using-SPDK-vhostuser-and-kata.md:235]: Nit: Missing space between `example` and left paren

File: docs/use-cases/using-SPDK-vhostuser-and-kata.md [hunk 2]
```diff
@@ -228,20 +228,41 @@ $ sudo mknod /var/run/kata-containers/vhost-user/block/devices/vhostblk0 b 241 0
...
+        "fileMode": 420,
+        "uid": 0,
+        "gid": 0
+      }
+    ]
+  }
+}
+```
+
+With `rootfs` provisioned under `bundle` directory, you can run your spdk container:
```
Comment (cmaf) [inline docs/use-cases/using-SPDK-vhostuser-and-kata.md:255]: s/spdk/SPDK/

**Review by cmaf — COMMENTED** [review]
@liubin a few minor suggestions

**Comment by liubin** [comment]
Thank you @cmaf , fixed.

**Review by lifupan — APPROVED** [review]

**Review by Tim-Zhang — APPROVED** [review]

**Pull request merged**